config/params.php

```diff
@@ -11,6 +11,7 @@
         'commands' => [
             'serve' => Serve::class,
         ],
-        'version' => '3.0'
+        'version' => '3.0',
+        'rebuildConfigs' => getenv('APP_ENV') === 'dev',
```

Comment:
It will work after https://github.com/yiisoft/composer-config-plugin/issues/120 is implemented. Right now it will read environment of where configs are being built, not runtime environment.

Reply:
Overall idea to move it to params is good.

Reply:
Make so: 
```php
'rebuildConfig' => static fn() => getenv('APP_ENV') === 'dev',
```